client/src/containers/Register/index.js

```diff
@@ -74,55 +98,71 @@ class Register extends PureComponent {
 
     render() {
         const { users } = this.props;
+        const {
+            name, lastname,
+            email, password
+        } = this.state.formData;
+
+        console.log(this.props)
```

Comment:
Unnecessary code ;)

Reply:
Yes, true! Thank you! Just forgot to delete the code from debug...